tox.ini

```diff
@@ -44,6 +44,7 @@ basepython = python3.10
 
 deps =
     wagtail>=5.1
+    .[testing]
```

Comment:
note to self: this could be 
`extras = testing` ([reference](https://tox.wiki/en/4.11.3/config.html#extras))

but this works

Reply:
Just followed what was in the `[testenv]` -> `deps`.

Reply:
yup. and that was good call to be consistent.

this is for me to follow up on ;)